test/runners.jl

```diff
@@ -322,3 +322,10 @@ end
         end
     end
 end
+
+@testset "outptut runner" begin
+    io = IOBuffer()
+    run_interactive(BinaryBuilderBase.preferred_runner()(mktempdir(); platform=Platform("x86_64", "linux"; libc="musl")), `/bin/bash -c "echo \$PATH"`, stdout=io)
+    s = String(take!(io))
+    @test !isempty(s)
```

Comment:
In https://github.com/JuliaPackaging/BinaryBuilderBase.jl/pull/197#issuecomment-1003012107 I used `echo $PATH` because that's what I had in my history, but for the test it may be better to print something we can check for sure, like
```suggestion
    run_interactive(BinaryBuilderBase.preferred_runner()(mktempdir(); platform=Platform("x86_64", "linux"; libc="musl")), `/bin/bash -c "echo hello world"`, stdout=io)
    s = String(take!(io))
    @test s == "hello world\n"
```